api/client/start.go

```diff
@@ -32,6 +32,7 @@ func (cli *DockerCli) forwardAllSignals(cid string) chan os.Signal {
 			}
 			if sig == "" {
 				fmt.Fprintf(cli.err, "Unsupported signal: %v. Discarding.\n", s)
+				continue
```

Comment:
@calavera  `signal.ParseSignal` translates a string to a signal,  I think we may don't need it here.